plugins/woocommerce-admin/client/tasks/fills/experimental-import-products/CardList.tsx

```diff
@@ -11,7 +11,7 @@ import './cards.scss';
 type Card = {
 	key: string;
 	title: string;
-	content: string;
+	content: string | JSX.Element;
 	before: JSX.Element;
 };
 
```

Comment:
I just noticed that the card title is still black when hovering.

<img width="1444" alt="WooCommerce _ Home _ Add Products - Import Version - Hover Effect" src="https://user-images.githubusercontent.com/4344253/166402031-afb7e962-a5af-44f7-837a-8d0a0ca8762b.png">

Reply:
The second screenshot from https://github.com/woocommerce/woocommerce/issues/32143 shows it as black. Do we need to change the color to blue on hover? 

cc @verofasulo

Reply:
@moon0326 Ah, sorry, I just checked again. It's actually blue now and I think we should change it to black.

![Screen Shot 2022-05-04 at 09 45 00](https://user-images.githubusercontent.com/4344253/166613015-6a2b45a7-5e4c-4ba7-b816-5ae9b4a41e79.png)

Reply:
@chihsuan You're right. I was actually looking at the stack. Updated in https://github.com/woocommerce/woocommerce/pull/32835/commits/8219510fc5dc050e758c499cd257d68ebfbde589